platforms/ide/ide-native/src/main/java/org/gradle/ide/visualstudio/internal/VisualStudioProjectConfiguration.java

```diff
@@ -35,47 +30,30 @@ public VisualStudioProjectConfiguration(DefaultVisualStudioProject vsProject, St
         this.binary = binary;
     }
 
-    @Input
```

Comment:
This type is part of the mutable model and is no longer used (or should be used) as a @Nested bean.